File: server/nodejs/src/secrets.ts
import dotenv from "dotenv";

dotenv.config({ path: ".env" });

// Server
export const PORT = process.env.PORT || 4000;
export const NODE_ENV = process.env.NODE_ENV;

// Tokens
export const ACCESS_TOKEN_SECRET = process.env.ACCESS_TOKEN_SECRET!;
export const REFRESH_TOKEN_SECRET = process.env.REFRESH_TOKEN_SECRET!;
export const PASSWORD_TOKEN_SECRET = process.env.PASSWORD_TOKEN_SECRET!;

// Email
export const EMAIL_HOST = process.env.EMAIL_HOST;
export const EMAIL_PORT = process.env.EMAIL_PORT;
export const EMAIL_ADDRESS = process.env.EMAIL_ADDRESS;
export const EMAIL_PASSWORD = process.env.EMAIL_PASSWORD;

File: server/python/app/secrets.py
import os
from dotenv import load_dotenv

load_dotenv()

NODE_ENV = os.getenv("NODE_ENV")
PORT = os.getenv("PORT")

ACCESS_TOKEN_SECRET = os.getenv("ACCESS_TOKEN_SECRET")
REFRESH_TOKEN_SECRET = os.getenv("REFRESH_TOKEN_SECRET")
PASSWORD_TOKEN_SECRET = os.getenv("PASSWORD_TOKEN_SECRET")

EMAIL_HOST = os.getenv("EMAIL_HOST")
EMAIL_NAME = os.getenv("EMAIL_NAME")
EMAIL_ADDRESS = os.getenv("EMAIL_ADDRESS")
EMAIL_PASSWORD = os.getenv("EMAIL_PASSWORD")
EMAIL_PORT = os.getenv("EMAIL_PORT")
EMAIL_SECURITY = os.getenv("EMAIL_SECURITY")

DATABASE_URL = os.getenv("DATABASE_URL")
